image_client/specify_db.py

```diff
@@ -2,10 +2,11 @@
 
 
 class SpecifyDb(DbUtils):
-    def __init__(self,db_config_class):
+    def __init__(self, db_config_class):
         self.specify_db_connection = super().__init__(
             db_config_class.USER,
             db_config_class.PASSWORD,
             db_config_class.SPECIFY_DATABASE_PORT,
             db_config_class.SPECIFY_DATABASE_HOST,
             db_config_class.SPECIFY_DATABASE)
+        print("initializing that thing")
```

Comment:
let's be a little more specific here, and it should also probably be a logger.debug or a logger.info.